components/03-modules/header/_header.scss

```diff
@@ -1,96 +1,61 @@
-$header__margin                : 4px 0 !default;
-$header__margin--xl            : 0 !default;
-$header__border-color          : $gray !default;
-$header__font-size             : $font-size-small !default;
-$header__button-size           : 48px !default;
-$header__icon-size             : 24px !default;
-$header__icon-padding          : 2px !default;
-$header__icon-color            : $gray-dark !default;
+$header__margin--extra-large           : 0 !default;
+$header__font-size                     : $font-size-small !default;
+$header__border-color                  : $gray-light !default;
+
+$header__wrapper-padding               : 0 !default;
+$header__wrapper-padding--medium       : 12px 0 !default;
+
+$header__logo-padding                  : 12px 0 12px $spacer !default;
+$header__logo-padding--medium          : 0 0 0 $spacer !default;
+
+$header__menu-trigger-size             : 48px !default;
+
+$header__buttons-width                 : calc(90% - 48px)  !default;
+$header__buttons-width--medium         : calc(4 * 48px + 3 * #{$spacer}) !default;
+$header__buttons-width--extra-large    : calc(4 * 48px + 3 * #{$spacer--medium}) !default;
+
+$header__icon-size                     : 24px !default;
+$header__icon-padding                  : 2px !default;
+$header__icon-color                    : $gray-darker !default;
+
+$header__search-wrapper-padding        : 0 0 0 24px !default;
+$header__search-wrapper-padding--medium: 0 24px !default;
```

Comment:
I'm just wondering why the spacing values are not the same as our global spacers... it's new design? The main spacer didn't change? sick...

Reply:
these are value from zeplin e.g. http://prntscr.com/hcrbl2

Reply:
it's true, it's sad :(